controllers/auth.go

```diff
@@ -782,6 +782,17 @@ func (c *ApiController) Login() {
 				resp = &Response{Status: "error", Msg: "Failed to link user account", Data: isLinked}
 			}
 		}
+		userProvider := &object.UserProvider{
```

Comment:
@souress what the reason to add this?

Reply:
FYI. fixed logic in latest commits

The User Provider table is needed to get additional information about the user, such as the name of the provider the user came from and his name in provider